services/storage/src/simcore_service_storage/modules/db/file_meta_data.py

```diff
@@ -94,6 +94,12 @@ def _list_filter_with_partial_file_id_stmt(
                 ),
             )
         )
+    else:
+        project_ids = user_or_project_filter.project_ids
```

Comment:
Q: do you have an upper limit in the number of `project_ids` you can pass? I would restrict it in the rest-api